### ISSUE TITLE
Feature Request: Increase Limit of 10 Pages

### ISSUE THREAD

**Issue by batman2999** [willemclarke/reads-good#1]
This little program is great!  Really saving my butt when moving from Readarr to LL. One small feature request... many lists are longer than 10 pages, is it complicated to increase this limit or is it an arbitrary limit?  

Thanks so much!

**Comment by willemclarke** [comment]
Hey, I'm surprised yet happy to see someone is getting some value from this little cobbled together CLI. As for the 10 page limit, it was just arbitrary (as I thought no one would use it, 10 pages felt like a responsible number to not pillage their servers). However, I can and will update this so you can supply any number. I'm away at the moment but it's a simple fix, I will report back here when I update it.

**Comment by Ty-Grr** [comment]
Hi @willemclarke 
Any update to this? am starting to use this and its great, would be good to use any number. potentially add delay as to not blitz their servers.

**Comment by willemclarke** [comment]
@joreilly02 @Ty-Grr Sorry for taking ages on this. I've pushed a [commit](https://github.com/willemclarke/reads-good/commit/74aad782adb5d71f71d640a7ea1cf72070f5c524) to handle your use-cases.

TLDR is:

- I now support a max of **100** pages. From looking on good reads, the largest list I found had 100 pages and has 75,707 books, so this should have you covered.
- Now that I'm supporting more pages, Ive added a delay of **3** seconds between each page, and **500ms** between each book per page.
- I tested on a listopia list that had three pages, it seemed to short circuit correctly when attempting the fourth page, let me know if its giving you any grief

Glad you're getting value out of it.
